performance-metrics/src/main.rs

```diff
@@ -150,10 +150,23 @@ impl PerformanceTest {
             metrics.push((self.func_ptr)(&self.control));
         }
 
-        let mean = (self.unit_adjuster)(mean(&metrics).unwrap());
-        let std_dev = (self.unit_adjuster)(std_deviation(&metrics).unwrap());
-        let max = (self.unit_adjuster)(metrics.clone().into_iter().reduce(f64::max).unwrap());
-        let min = (self.unit_adjuster)(metrics.clone().into_iter().reduce(f64::min).unwrap());
+        /*
+         * As ethr is not stable on arm, there may get no output from
+         * the test, in this case, an invalid value of "-1.0" is
+         * injected, so we need eliminate these incorrect values here
+         * before use the vector.
+         */
+        let mut fixed_metrics = Vec::new();
+        for i in metrics {
+            if i >= 0.0 {
+                fixed_metrics.push(i);
+            }
```

Comment:
Can we mark this latency test for `x86_64` only for now? Let's see whether we can have better solutions other than the hardcoded workaround which I am afraid it would easily break or generate misleading numbers.

Reply:
My bad, I forgot masking x86 here.